src/Campaigns/ServiceProvider.php

```diff
@@ -119,6 +119,11 @@ private function setupCampaignForms()
             Hooks::addAction('admin_enqueue_scripts', DonationFormsAdminPage::class, 'loadScripts');
         }
 
+        // Temp solution to load "donations" and "revenue" columns on the "Forms" tab
+        if ( ! defined('GIVE_IS_ALL_STATS_COLUMNS_ASYNC_ON_ADMIN_FORM_LIST_VIEWS')) {
+            define('GIVE_IS_ALL_STATS_COLUMNS_ASYNC_ON_ADMIN_FORM_LIST_VIEWS', false);
```

Comment:
Can you please give me a bit of context here? This is the first time I have seen this.

Reply:
Resolved in 072253e36369b1723d7bcced244842fb4b6fc7df